page-object/pages/SearchPage.ts

```diff
@@ -0,0 +1,68 @@
+import { Locator, Page } from "@playwright/test";
+import { BasePage } from "../BasePage";
+
+
+export class SearchPage extends BasePage {
+searchPageLocator: any;
+  readonly searchField: Locator;
+  readonly filterByDateBtn: Locator;
+  readonly filterByViewedBtn: Locator;
+  readonly filterByCreatorBtn: Locator;
+  readonly filterByDatasetSizeBtn: Locator;
```

Comment:
Suggest to change name to smth like DatasetSmallSizeBtn, as there are 3 dataset buttons with different sizes